vispy/app/base.py

```diff
@@ -186,6 +188,10 @@ def _vispy_mouse_press(self, **kwargs):
         return ev
 
     def _vispy_mouse_move(self, **kwargs):
+        if default_timer() - self._last_time < .01:
```

Comment:
I think this will actually be too slow on some displays (e.g., 144 Hz). But probably good enough for now.